simapp/app_v2.go

```diff
@@ -144,7 +144,7 @@ func NewSimApp(
 				// STAKING
 				//
 				// For provinding a different validator and consensus address codec, add it below.
-				// By default the staking module uses the bech32 prefix provided in the auth config,
+				// By default the staking module uses the hex prefix provided in the auth config,
```

Comment:
This comment needs update. There aren't hex, valoper and valcons prefixes/infixes

Reply:
Done, thanks